ci/repokitteh/modules/newpr.star

```diff
@@ -26,8 +34,10 @@ def should_message_newcontributor(action, issue_number):
 def send_newcontributor_message(sender):
   github.issue_create_comment(NEW_CONTRIBUTOR_MESSAGE % sender)
 
-def _pr(action, issue_number, sender, config):
+def _pr(action, issue_number, sender, config, draft):
   if should_message_newcontributor(action, issue_number):
     send_newcontributor_message(sender)
+  if draft:
```

Comment:
this will happen on each pr event. i think you want to gate it on `action == "opened"`. see https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request